research/dmri_hippo/run.py

```diff
@@ -93,8 +93,8 @@ def augmentation_experiment(
         context=context,
         max_iterations=100000,
         max_training_time=max_training_time,
-        preload_training_data=False,
-        preload_validation_data=False,
+        preload_training_data=True,
+        preload_validation_data=True,
```

Comment:
Think it is still okay to preload data if we are running multiple folds on a single CC node?

Reply:
I think it should be okay. It's only pre-loaded into RAM and our current utilization is around 20% on a job with 16GB allocated:
https://wandb.ai/efirdc/dmri-hippo-seg-v3/runs/1g9fgjb4/system?workspace=user-efirdc
We can always request more if we need it since I think they have tuns.